sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOverrides.scala

```diff
@@ -2362,6 +2362,20 @@ object GpuOverrides {
         override def convertToGpu(lhs: Expression, rhs: Expression): GpuExpression =
           GpuArrayContains(lhs, rhs)
       }),
+    expr[SortArray](
+      "Returns a sorted array with the input array and the ascending / descending order",
+      ExprChecks.binaryProjectNotLambda(
+        TypeSig.ARRAY.nested(TypeSig.commonCudfTypes + TypeSig.DECIMAL + TypeSig.NULL),
+        TypeSig.ARRAY.nested(TypeSig.all),
+        ("array", TypeSig.ARRAY.nested(TypeSig.commonCudfTypes + TypeSig.DECIMAL + TypeSig.NULL),
+            TypeSig.ARRAY.nested(TypeSig.all)),
+        ("ascendingOrder", TypeSig.BOOLEAN, TypeSig.BOOLEAN)),
+      (in, conf, p, r) => new BinaryExprMeta[SortArray](in, conf, p, r) {
+        override def convertToGpu(lhs: Expression, rhs: Expression): GpuExpression = {
```

Comment:
If we don't support literal arrays we have to tag it here so we fall back to the CPU.

Reply:
I think we can support literal arrays. And I made up the corresponding test.

Reply:
What you wrote is good enough for me. The test is likely not testing what we expect it to, but the code looks like it should work fine so I am OK with it.